### DISCUSSION TITLE
[Feature] Add Manual Photo Download Feature for Immich App

### DISCUSSION THREAD

**Discussion by WildBrianNL** [immich-app/immich#7747] [Feature Request]
### The feature

I'm using Immich as an alternative to Google Photos. I recently switched to a new phone, and would love a feature to manually download all photos back to my phone (in bulk). 

This way I don't have to use Syncthing anymore.  

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#5068
